packages/flutter/lib/src/painting/box_border.dart

```diff
@@ -538,15 +578,25 @@ class Border extends BoxBorder {
       }
     }
 
+    // Allow painting non-uniform borders if the color is uniform.
+    //
+    // See also:
+    //  * <https://pub.dev/packages/non_uniform_border>, a package that
```

Comment:
Can you move this "See also" to a real doc comment so people can see it?

Reply:
Done, check if you prefer in Border or paint (probably paint, right?).